packages/jellyfish-api-core/__tests__/category/loan/takeLoan.test.ts

```diff
@@ -808,3 +808,485 @@ describe('takeloan failed', () => {
     await expect(promise).rejects.toThrow(`Vault with id ${emptyVaultId} has no collaterals`)
   })
 })
+
+describe('takeLoan with 50% DUSD or DFI collaterals', () => {
+  const fortCanningRoadHeight = 128
+  beforeEach(async () => {
+    const startFlags: StartFlags[] = [{ name: 'fortcanningroadheight', value: fortCanningRoadHeight }]
```

Comment:
after `fortcanningroadheight`, some tests from old tests will fail.
so add the flag to `RegTestContainer` and refactor tests.

Reply:
and for tests that check for previous HF behaviors, you can filter out `fortCanningRoadHeight` flag